DOC-3356: New feature documentation for `tinymceai`. (#3976)
tinymce/tinymce-docs: modules/ROOT/pages/tinymceai-actions-plugin.adoc

= TinyMCE AI Quick Actions

:navtitle: Quick Actions
:description: quick actions feature for TinyMCE AI plugin
:description_short: quick actions feature
:keywords: AI, quick actions, actions, tinymceai

> Quick actions streamline routine content transformations by offering one-click AI-powered suggestions directly within the editor. You can also ask questions about your selected text in AI chat to get instant AI insights and analysis. This feature enhances speed, relevance, and usability, particularly for repeatable or simple tasks. The feature comes with an easy-to-use window interface but can also act as a conversation starter with the xref:tinymceai-chat.adoc[Chat].

Reviewer: ```suggestion
Quick actions streamline routine content transformations by offering one-click AI-powered suggestions directly within the editor. This feature enhances speed, relevance, and usability, particularly for repeatable or simple tasks. The feature comes with an easy-to-use window interface but can also act as a conversation starter with the xref:tinymceai-chat.adoc[Chat].
```
that sentence is about chat, not quick actions.

Author: @metricjs 

Removed:
- The sentence "You can also ask questions about your selected text in AI chat to get instant AI insights and analysis." from the Quick actions description
- Kept it focused on Quick actions functionality only

Updated in https://github.com/tinymce/tinymce-docs/pull/3976/commits/7bb949474064a56214a6d0743076fb10e7e485d9

[[demo]]
== Demo

liveDemo::tinymceai[]

[[overview]]
== Overview

Actions are fast, stateless operations that transform content directly. Unlike Reviews that provide suggestions, Actions immediately modify your content based on the selected operation.

> **When to use Actions vs Reviews:** Use Actions when you need to transform specific text content (fix grammar, translate, adjust tone). Use Reviews when you need to analyze entire documents for quality improvements and get suggestions without automatically changing the content.

Reviewer: Same as before - Instead of "specific text content" should we say something like a selection or a small section or something? Since Quick Actions work on selections specifically.

Author: @metricjs 

Updated:
- Changed "transform specific text content" to "transform a selection or small section of text"
- Accurately reflects that Quick Actions work on selections specifically

Updated in https://github.com/tinymce/tinymce-docs/pull/3976/commits/7bb949474064a56214a6d0743076fb10e7e485d9

image::https://placehold.net/default.png[TinyMCE AI quick actions showing AI-powered suggestions and Q&A functionality]

[[integration]]
== Integration

To start using the Quick Actions feature, first load the TinyMCE AI plugin in your editor configuration. xref:tinymceai-integration.adoc[Learn more about installing and enabling AI features].

> Then, you can add the menu that opens the list of Quick Actions (`tinymceai-actions`) image:https://placehold.net/default.png[Quick Actions icon,24px] to your main toolbar and/or balloon toolbar configurations. To learn more about toolbar configuration, refer to the xref:toolbar-configuration-options.adoc[toolbar configuration] guide.

Reviewer: Is "balloon toolbar" the equiv of our context toolbars? I assume so. Also want to check if we add the AI features to the selection toolbar by default if the plugin is enabled - Idk if we're doing that currently

Author: @metricjs 

Updated:
- Changed "balloon toolbar" to "context toolbar (selection toolbar)" to clarify they're equivalent
- Added NOTE confirming AI features are added to selection toolbar by default when plugin is enabled
- Added that `tinymceai_context_toolbar` configuration option controls which features appear
- Added examples showing how to configure specific features (e.g., `'showai aiquickactions aireview'` vs. `'showai aireview'`)

Updated in https://github.com/tinymce/tinymce-docs/pull/3976/commits/7bb949474064a56214a6d0743076fb10e7e485d9

    return fetch('/api/token').then(response => response.text());
  },
  
  // Adding Quick Actions to the main editor toolbar.
  toolbar: [
    // The main Quick Actions button
    'tinymceai-actions',
    
    // Two individual actions
    'ask-ai',
    'improve-writing',
    
    // Whole action category
    'translate',
    
    // ... other toolbar items
  ],
  
  // Adding Quick Actions to the balloon toolbar. Since some of the actions are selection-sensitive,
  // accessing them might be easier for users using this kind of toolbar.
  quickbars_selection_toolbar: 'tinymceai-actions | ask-ai | improve-writing | translate'
});
----

[[types-of-actions]]
== Types of actions

There are two types of actions available in the quick actions feature:

* Some actions, for instance, "Ask AI" or "Summarize", lead to the xref:tinymceai-chat.adoc[Chat] interface with selected text added as context. The former will just open the Chat and allow you to start typing your message. The latter, however, will not only open the Chat but also start the conversation for your current editor selection right away, and expect a summary of that selection from the AI.

> * Executing other actions like "Continue writing" or "Make shorter" will open the window interface conveniently right next to your selection and present the answers from the AI for you to accept or reject them.

Reviewer: pretty sure our UI ended up being a bit different for this

Author: @metricjs 

Updated after verifying against plugin code:
- Simplified description to match actual implementation
- Updated example configuration to match actual working code:
  - Toolbar includes all three main buttons: `showai aiquickactions aireview`
  - Context toolbar configured with `tinymceai_context_toolbar: 'showai aiquickactions aireview'`
  - Added menu configuration examples: `tinymceai_quickactions_menu` and `tinymceai_chat_commands_menu`
  - Shows correct menu structure with sub-menus (e.g., chat commands sub-menu)

Updated in https://github.com/tinymce/tinymce-docs/pull/3976/commits/7bb949474064a56214a6d0743076fb10e7e485d9

Keep in mind that you can xref:tinymceai-actions-plugin.adoc#custom-actions[add custom actions] to the list and xref:tinymceai-actions-plugin.adoc#removing-default-actions[remove defaults].

Here's the full list of available actions:

* `'ask-ai'`
* **"Chat commands"** category (`'chat-commands'`)
  ** `'explain'`
  ** `'summarize'`
  ** `'highlight-key-points'`
* `'improve-writing'`
* `'continue'`
* `'fix-grammar'`
* **"Adjust length"** category (`'adjust-length'`)
  ** `'make-shorter'`
  ** `'make-longer'`
* **"Change tone"** category (`'change-tone'`)
  ** `'make-tone-casual'`
  ** `'make-tone-direct'`
  ** `'make-tone-friendly'`
  ** `'make-tone-confident'`
  ** `'make-tone-professional'`
* **"Translate"** category (`'translate'`)
  ** `'translate-to-english'`
  ** `'translate-to-chinese'`
  ** `'translate-to-french'`
  ** `'translate-to-german'`
  ** `'translate-to-italian'`
  ** `'translate-to-portuguese'`
  ** `'translate-to-russian'`

[[custom-actions]]
== Custom Actions

> The `tinymceai_actions_extraCommands` configuration property allows you to add new commands to the Quick actions feature. Below, you will find an example of three extra actions added to the user interface: two of them open the quick actions window, but the last one interacts with AI chat. Learn more about xref:tinymceai-actions-plugin.adoc#types-of-actions[types of actions].

Reviewer: tinymceai_custom_commands

Author: @metricjs 

Updated:
- Changed property name from `tinymceai_actions_extraCommands` to `tinymceai_custom_commands`
- Updated type values from uppercase (`'ACTION'`, `'CHAT'`) to lowercase (`'action'`, `'chat'`)
- Removed `id` field
- Updated model example
- Enhanced description to explain `displayedPrompt`, `prompt`, `type`, and `model` fields

Updated in https://github.com/tinymce/tinymce-docs/pull/3976/commits/7bb949474064a56214a6d0743076fb10e7e485d9

[source,js]
----
tinymce.init({
  selector: '#editor',
  plugins: 'tinymceai',
  toolbar: 'tinymceai-actions',
  tinymceai_token_provider: function() {
    return fetch('/api/token').then(response => response.text());
  },
  tinymceai_actions_extraCommands: [
    {
      id: 'add-quote-from-famous-person',
      displayedPrompt: 'Add a quote from a famous person',
      prompt: 'Add a quote from a known person, which would make sense in the context of the selected text.',
      type: 'ACTION',
      model: 'agent-1'
    },
    {
      id: 'summarize-in-bullet-points',
      displayedPrompt: 'Summarize in 5 bullet points',
      prompt: 'Summarize the selected text in 5 bullet points.',
      type: 'CHAT'
    },
    {
      id: 'include-more-sarcasm',
      displayedPrompt: 'Rewrite adding more sarcasm',
      prompt: 'Rewrite using a sarcastic tone.',
      type: 'ACTION',
      model: 'agent-1'
    }
    
    // ... More commands ...
  ]
});
----

[[removing-default-actions]]

> == Removing default actions

Reviewer: we let people configure the list instead of them having to specify which to remove. option is. tinymceai_quickactions_menu for the main menu, then any sub menus have their own option (tone, length, etc)

Author: @metricjs 

Updated:
- Changed section title from "Removing default actions" to "Configuring Quick Actions menu"
- Removed reference to non-existent `tinymceai_actions_removeCommands` property
- Updated to explain configuration approach:
  - `tinymceai_quickactions_menu` for main menu
  - Sub-menu options: `tinymceai_chat_commands_menu`, `tinymceai_adjust_length_menu`, `tinymceai_change_tone_menu`, `tinymceai_translate_menu`
- Updated code example to show configuring the list rather than removing items

Updated in https://github.com/tinymce/tinymce-docs/pull/3976/commits/7bb949474064a56214a6d0743076fb10e7e485d9

= TinyMCE AI Quick Actions
:navtitle: Quick Actions
:pluginname: TinyMCE AI
:plugincode: tinymceai
:description: Quick actions feature for TinyMCE AI plugin
:description_short: quick actions feature
:keywords: AI, quick actions, actions, tinymceai
:page-role: -toc

> Quick actions simplify routine content transformations by offering one-click AI-powered suggestions directly within the editor. This feature enhances speed, relevance, and usability, particularly for repeatable or simple tasks. The feature comes with a window interface for most actions; some actions (such as Explain or Summarize) open the xref:tinymceai-chat.adoc[Chat] instead and start the conversation with a pre-filled prompt.

Reviewer: ```suggestion
Quick actions simplify routine content transformations by offering one-click AI-powered suggestions directly within the editor. This feature enhances speed, relevance, and usability, particularly for repeatable or simple tasks. The plugin feature comes with a review interface for most AI suggestions; some actions (such as Explain or Summarize) open the xref:tinymceai-chat.adoc[Chat] instead and start the conversation with a pre-filled prompt. The API includes the ability to call predefined and optimized system actions, or to define your own.
```

> The Quick Actions feature is available as a plugin UI and through the Actions API. This page covers both:

Reviewer: ```suggestion
The Quick Actions feature is available as part of the `+tinymceai+` plugin for {productname} and through the Actions API. This page covers both:
```

* Plugin sections describe user-facing features and configuration (xref:tinymceai.adoc#options-quickactions[Quick Actions configuration]).
* API sections cover REST API access (starting at xref:tinymceai-actions-plugin.adoc#actions-api[Actions API]).

[[demo]]
== Demo

liveDemo::tinymceai-actions[]

> Actions are fast, stateless operations that transform content. Unlike Reviews that provide suggestions for review, Actions run a transformation and display the result in a preview interface where users can accept or reject before changes are applied.

Reviewer: this comparison doesn't make sense?

Reviewer: nor the vs section below

Reviewer: per comments on the Review page

Author: Updated in [this commit](https://github.com/tinymce/tinymce-docs/pull/3976/commits/acbdd33695b9a042dd8b9b76e1416de2a3b365f6) to reframe comparison.

**When to use Actions vs Reviews?**

* *Actions*: Transform a selection or small section of text (fix grammar, translate, adjust tone).
* *Reviews*: Analyze entire documents for quality improvements and get suggestions without automatically changing the content.

image::tinymceai/tinymce-ai-quick-actions-toolbar-primary-menu-with-chat-commands.png[{pluginname} quick actions showing AI-powered suggestions and Q&A functionality,width=80%]

[[integration]]
== Integration

To start using the Quick Actions feature, first load the {pluginname} plugin in the editor configuration. See xref:tinymceai.adoc[Plugin Reference] for installation and enabling AI features.

Then, add the menu that opens the list of Quick Actions (`tinymceai-quickactions`) image:icons/ai-prompt.svg[Quick Actions icon,24px] to the main toolbar and/or context toolbar (selection toolbar) configurations. 

[NOTE]
====

> AI features are added to the selection toolbar by default when the plugin is enabled. Use the toolbar configuration options to customize which AI features appear.

Reviewer: wait. they're added to the normal toolbar by default. if "selection toolbar" = quickbars, pretty sure that needs to be added manually.

Author: Good catch - updated [this commit](https://github.com/tinymce/tinymce-docs/pull/3976/commits/4f8011302f7f818f707bb3c21ecad24539527d06) to fix this issue.

The final example configuration looks as follows:

[source,js]
----
tinymce.init({
  selector: '#editor',
  plugins: 'tinymceai',
  toolbar: 'undo redo tinymceai-chat tinymceai-quickactions tinymceai-review | styles | bold italic underline strikethrough | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image',
  tinymceai_quickactions_menu: [
    'ai-quickactions-chat-prompts',
    'ai-quickactions-improve-writing',
    'ai-quickactions-continue-writing',
    'ai-quickactions-check-grammar',
    'ai-quickactions-change-length',
    'ai-quickactions-change-tone',
    'ai-quickactions-translate'
  ],
  
  // Configure a sub-menu (e.g., chat commands)
  tinymceai_quickactions_chat_prompts: [
    'ai-chat-explain',
    'ai-chat-summarize',
    'ai-chat-highlight-key-points'
  ],

>   tinymceai_sidebar_type: 'static',

Reviewer 2: `tinymceai_sidebar_type` option defaults to `static` so it's not necessary to repeat that

  // Required for authentication
  tinymceai_token_provider: () => {
    return fetch('/api/token').then(r => r.text()).then(token => ({ token }));
  }
});
----

[[types-of-actions]]
== Types of actions

There are two types of actions available in the quick actions feature:

* Chat actions, such as "Explain" or "Summarize", open the xref:tinymceai-chat.adoc[Chat] interface with the selected text added as context and automatically start the conversation with a pre-filled prompt.

> * Preview actions, such as "Continue writing", "Make shorter", "Improve writing", or "Fix grammar", display the AI response in a preview interface where users can review and accept or reject the suggested changes.

Reviewer: not sure how to improve the wording, but "preview interface" feels wrong to me. it's the same interface for reviewing suggestions that we use with chat basically??

Reviewer 3: I'd probably go with something more like this:

```suggestion
* Preview actions such as _Continue writing_, _Make shorter_, _Improve writing_, and _Fix grammar_ show a preview of your document where suggested edits appear so you can review them and apply or skip each change.
```

Feel free to remove the italics if it's not part of our convention.

Custom actions let integrators define whether each action opens Chat or shows a preview. See xref:tinymceai-actions-plugin.adoc#custom-actions[Custom Actions].

[[default-actions]]
== Default Actions

By default, the Quick Actions feature includes several built-in actions that speed up the content editing process. All Quick Actions can be accessed through the menu button image:icons/ai-prompt.svg[Quick Actions icon,24px] (`tinymceai-quickactions`) or individually when selected by the integrator in the xref:toolbar-configuration-options.adoc[editor toolbar configuration]. Whole action categories can be added to the toolbar as well.

Custom actions can be added to the list, and default actions can be removed or reordered by configuring `tinymceai_quickactions_menu`. See xref:tinymceai-actions-plugin.adoc#configuring-quick-actions-menu[Configuring Quick Actions menu].

[[quick-actions-reference-table]]

> The following table lists built-in Quick Actions. The **Editor command** column is the menu label. The **Menu or control ID** column lists values used in `tinymceai_quickactions_menu`, `tinymceai_quickactions_chat_prompts`, `tinymceai_quickactions_change_tone_menu`, and the toolbar when adding individual controls (see xref:tinymceai-actions-plugin.adoc#configuring-quick-actions-menu[Configuring Quick Actions menu]). The **System `actionName` or quick-action prompt id** column is the service identifier: for preview actions it is the `actionName` path segment in `+/v1/actions/system/{actionName}/calls+` (see xref:tinymceai-actions-plugin.adoc#actions-api[Actions API]); for Chat commands it is the built-in quick-action prompt id (`explain`, `summarize`, `highlight-key-points`) used when opening xref:tinymceai-chat.adoc[Chat] with a pre-filled prompt—not a system `actionName` and not sent on the system actions path.

Reviewer: ```suggestion
The following table lists built-in Quick Actions. 

- The **Editor command** column is the menu label. 
- The **Menu or control ID** column lists values used in `tinymceai_quickactions_menu`, `tinymceai_quickactions_chat_prompts`, `tinymceai_quickactions_change_tone_menu`, and the toolbar when adding individual controls (see xref:tinymceai-actions-plugin.adoc#configuring-quick-actions-menu[Configuring Quick Actions menu]). 
- The **System `actionName`** column is the service identifier used as the `actionName` path segment in `+/v1/actions/system/{actionName}/calls+` (see xref:tinymceai-actions-plugin.adoc#actions-api[Actions API]).
```
easier to read as a list? 
edit: I think we can simplify the last point to just the API identifier too? based on a slack thread, I don't think we need to include that detail?

[cols="1,3,2,3,1,1",options="header"]
|===

> |Editor command |Menu or control ID |System `actionName` or quick-action prompt id |Description |Editor UI |API

Reviewer: ```suggestion
|Editor command |Menu or control ID |System `actionName`  |Description |Editor UI |API
```
simplifying in alignment with my other comments

`ai-quickactions-translate-swedish` +
`ai-quickactions-translate-german` +
`ai-quickactions-translate-japanese` +
`ai-quickactions-translate-portuguese` +
`ai-quickactions-translate-korean` +
`ai-quickactions-translate-italian` +
`ai-quickactions-translate-chinese`
a|
`translate`
|Convert content between languages. Request body uses `args: { language: string }` with values such as `english`, `spanish`, `russian`, `swedish`, `german`, `japanese`, `portuguese`, `korean`, `italian`, or `chinese`. Configure the editor submenu with `tinymceai_languages`. |✓ |✓
|===

[NOTE]
====
* A checkmark in the **Editor UI** column means the action can appear when included in `tinymceai_quickactions_menu` (and related sub-menu options such as `tinymceai_quickactions_chat_prompts`).
* A checkmark in the **API** column means the action is invoked through the system Actions API path `+/v1/actions/system/{actionName}/calls+`. An em dash means the command is Chat-based and uses the Conversations API instead.
====

For system action endpoints, schemas, and streaming details, see the https://tinymceai.api.tiny.cloud/docs#tag/Actions[Actions API] reference.

[[custom-actions]]
== Custom Actions

> The `tinymceai_quickactions_custom` configuration property allows adding new commands to the Quick actions feature. The items can be type `action` (a quick action that goes straight to the preview rendering) or type `chat` (opens the chat interface). The `title` is used in the menu items, while the `prompt` is what is actually sent to the AI. The `model` is required for `action` type commands. Learn more about xref:tinymceai-actions-plugin.adoc#types-of-actions[types of actions].

Reviewer: "preview rendering" again, if we change the earlier instance

Reviewer 3: I agree with @metricjs 

I don't think we need to be so specific about the preview rendering as an action. Yes, we render a preview iframe to get the effect of a preview but it is just an implementation detail. All the user should care about is that they are shown a preview of their editor content with AI suggestions included and that they have a way for the user to either apply or skip those changes before finalizing.

I would probably rephrase that more like:
> The items can be of type `action` (a quick action that shows a preview of the editor content with AI suggestions so you can review them before applying) or type `chat` (which opens the chat interface for a conversation related to the prompt and selection).

Not 100% sure about the wording but something more like that I think.

Author: Updated to reflect suggested improvement thanks Kim, see https://github.com/tinymce/tinymce-docs/pull/3976/commits/91fa046c658c22504acc996904e3bb9cc38b298a

Sub-menus have their own configuration options:

* `tinymceai_quickactions_chat_prompts`: Controls items in the **"Chat commands"** sub-menu
* `tinymceai_quickactions_change_tone_menu`: Controls items in the **"Change tone"** sub-menu
* `tinymceai_languages`: Controls languages in the **"Translate"** sub-menu

[source,js]
----
tinymce.init({
  selector: '#editor',
  plugins: 'tinymceai',
  toolbar: 'tinymceai-quickactions',
  // Required for authentication
  tinymceai_token_provider: () => {
    return fetch('/api/token').then(r => r.text()).then(token => ({ token }));
  },
  // Configure the main Quick Actions menu
  tinymceai_quickactions_menu: [
    'ai-quickactions-chat-prompts',
    'ai-quickactions-improve-writing',
    'ai-quickactions-continue-writing',
    'ai-quickactions-check-grammar',
    'ai-quickactions-change-length',
    'ai-quickactions-change-tone',
    'ai-quickactions-translate',
    'ai-quickactions-custom'
  ],
  // Configure a sub-menu (e.g., chat commands)
  tinymceai_quickactions_chat_prompts: [
    'ai-chat-explain',
    'ai-chat-summarize',
    'ai-chat-highlight-key-points'
  ]
});
----

[[actions-api]]
== Actions API

> The Quick Actions plugin feature is built on top of the Actions API, which provides REST API access to action functionality. Actions are fast, stateless operations that transform content. Unlike Reviews that provide suggestions for review, Actions run a transformation and display the result in a preview interface where users can accept or reject before changes are applied.

Reviewer: comparison with Review again

**When to use Actions vs Reviews:** Use Actions when transforming a selection or small section of text (fix grammar, translate, adjust tone). Use Reviews when analyzing entire documents for quality improvements and getting suggestions without automatically changing the content.

> Unlike conversations, actions do not remember previous interactions. Each action is independent and focused on a single task. Actions use streaming output with Server-Sent Events for real-time feedback as results are generated.

Reviewer: feels like the first sentence should be early in the document, since it applies to plugin and api. this is what we mean be "stateless" or however we worded it earlier tbh, but this clarification could do to be wherever we say that

Author: Updated in https://github.com/tinymce/tinymce-docs/pull/3976/commits/b01fa11fe0dc31093c0314edfac0626570b1ee0f

Unlike system actions that use predefined identifiers, custom actions use a unified endpoint where the transformation behavior is defined through a prompt parameter. See https://tinymceai.api.tiny.cloud/docs#tag/Actions[Actions API] for the custom actions endpoint and implementation details.

Custom actions require the `ai:actions:custom` permission in the JWT token.

[[actions-streaming]]
=== Streaming Responses

Actions use Server-Sent Events (SSE) for real-time streaming results. See the xref:tinymceai-streaming.adoc[Streaming Responses guide] for detailed implementation information.

[[actions-api-reference]]
=== API Reference

The https://tinymceai.api.tiny.cloud/docs#tag/Actions[Actions API] reference (interactive OpenAPI documentation) is the full source for system and custom action endpoints, request and response schemas, authentication, worked examples, and streaming action responses.

[[related-features]]
== Related Features

* xref:tinymceai-chat.adoc[AI chat]: For interactive discussions with document analysis and context.

> * xref:tinymceai-review-plugin.adoc[AI review]: For content quality analysis and improvement suggestions.

Reviewer: should the feature names be capitalised?

```suggestion
* xref:tinymceai-chat.adoc[AI Chat]: For interactive discussions with document analysis and context.
* xref:tinymceai-review-plugin.adoc[AI Review]: For content quality analysis and improvement suggestions.
```

* Preview actions, such as "Continue writing", "Make shorter", "Improve writing", or "Fix grammar", display the AI response in a preview interface where users can review and accept or reject the suggested changes.

Custom actions let integrators define whether each action opens Chat or shows a preview. See xref:tinymceai-actions-plugin.adoc#custom-actions[Custom Actions].

[[default-actions]]
== Default Actions

By default, the Quick Actions feature includes several built-in actions that speed up the content editing process. All Quick Actions can be accessed through the menu button image:icons/ai-prompt.svg[Quick Actions icon,24px] (`tinymceai-quickactions`) or individually when selected by the integrator in the xref:toolbar-configuration-options.adoc[editor toolbar configuration]. Whole action categories can be added to the toolbar as well.

Custom actions can be added to the list, and default actions can be removed or reordered by configuring `tinymceai_quickactions_menu`. See xref:tinymceai-actions-plugin.adoc#configuring-quick-actions-menu[Configuring Quick Actions menu].

[[quick-actions-reference-table]]
The following table lists built-in Quick Actions. 

> - The **Editor command** column is the menu label. 

Reviewer 4: I wonder if we should use other terminology here since command might be confused with editor commands like the execCommand type of command.

Author: We made similar updates to the other feature pages, so ill update this to reflect the same changes for `commands`. For more info on changes see https://github.com/tinymce/tinymce-docs/pull/3976/commits/0ad6f2a64d8dbe9ea63ecabf343ee51bb10f36bb

- The **Menu or control ID** column lists values used in `tinymceai_quickactions_menu`, `tinymceai_quickactions_chat_prompts`, `tinymceai_quickactions_change_tone_menu`, and the toolbar when adding individual controls (see xref:tinymceai-actions-plugin.adoc#configuring-quick-actions-menu[Configuring Quick Actions menu]). 

> - The **System `actionName` or quick-action prompt id** column is the service identifier: for preview actions it is the `actionName` path segment in `+/v1/actions/system/{actionName}/calls+` (see xref:tinymceai-actions-plugin.adoc#actions-api[Actions API]); for Chat commands it is the built-in quick-action prompt id (`explain`, `summarize`, `highlight-key-points`) used when opening xref:tinymceai-chat.adoc[Chat] with a pre-filled prompt—not a system `actionName` and not sent on the system actions path.

Reviewer 4: I wonder if we need to go into that low detail here. I guess we need to explain what the difference is between a system action and a chat command.

Author: Simplified the details here as requested, see https://github.com/tinymce/tinymce-docs/pull/3976/commits/0ad6f2a64d8dbe9ea63ecabf343ee51bb10f36bb for more details on updates if needed.
